release.js

```diff
@@ -0,0 +1,121 @@
+import chalk from 'chalk';
+import esMain from 'es-main';
+import fs from 'fs-extra';
+import inquirer from 'inquirer';
+
+import {exec} from './scripts.js';
+
+const currentVersion = (
+  await fs.readJson(new URL('./package.json', import.meta.url))
+).version;
+
+const getNewVersion = async () => {
+  const versionParts = currentVersion.split('.').map((x) => Number(x));
+  const newVersions = [
+    `${versionParts[0] + 1}.0.0`,
+    `${versionParts[0]}.${versionParts[1] + 1}.0`,
+    `${versionParts[0]}.${versionParts[1]}.${versionParts[2] + 1}`
+  ];
+
+  const answers = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'newVersion',
+      message: 'How should we bump the version?',
+      choices: [
+        {
+          name: `Major (${newVersions[0]})`,
+          value: newVersions[0]
+        },
+        {
+          name: `Minor (${newVersions[1]})`,
+          value: newVersions[1]
+        },
+        {
+          name: `Patch (${newVersions[2]})`,
+          value: newVersions[2]
+        }
+      ],
+      default: 2
+    }
+  ]);
+
+  return answers.newVersion;
+};
+
+const doVersionBump = async (newVersion) => {
+  for (const f of ['./package.json', './package-lock.json']) {
+    const filepath = new URL(f, import.meta.url);
+    const data = await fs.readJson(filepath);
+    data.version = newVersion;
+    await fs.writeJson(filepath, data, {spaces: 2});
+  }
+};
+
+const getChanges = () => {
+  const changes = exec(`git log --pretty=reference v${currentVersion}..main`)
```

Comment:
I don't have a local `head` branch (can't push to it anyway) so this didn't work for me. This will:

```suggestion
  const changes = exec(`git log --pretty=reference v${currentVersion}..HEAD`)
```